### PR TITLE
Add branding color quality check

### PR DESCRIPTION
**Pull request by razzeee**

### CODE REVIEW

File: backend/alembic/versions/248a867312b4_add_branding_quality_guideline.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            sa.Column("needed_to_pass_since", sa.Date(), nullable=False),
+            sa.Column("read_only", sa.Boolean(), nullable=False),
+            sa.Column("guideline_category_id", sa.String(), nullable=False),
+            sa.Column("show_on_fullscreen_app", sa.Boolean(), nullable=False),
+            sa.Column("order", sa.Integer(), nullable=False),
+        ),
+        [
+            {
+                "id": "general-has-primary-branding",
+                "url": "https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#brand-color",
```
Comment (razzeee) [inline backend/alembic/versions/248a867312b4_add_branding_quality_guideline.py:39]: We likely want to redo the docs here and maybe change the headline and thus this link

Comment (bertob) [reply backend/alembic/versions/248a867312b4_add_branding_quality_guideline.py:39]: "Brand color" sounds fine to me :+1:

Comment (razzeee) [reply backend/alembic/versions/248a867312b4_add_branding_quality_guideline.py:39]: I think it should be more self explanatory? It feels a bit displaced next to the other ones

Comment (razzeee) [reply backend/alembic/versions/248a867312b4_add_branding_quality_guideline.py:39]: More like `has primary branding colors` (similar to `At least one screenshots`, that's the vibe I'm after)

Comment (razzeee) [reply backend/alembic/versions/248a867312b4_add_branding_quality_guideline.py:39]: I think the appstream docs touch on that briefly, but we could too

**Pull request merged**